### ISSUE TITLE
Callback /event when formly finished rendering form?

### ISSUE THREAD

**Issue by the-clone** [ngx-formly/ngx-formly#2762]
We develop a large, form-based application. Sometimes there can be hundreds of form fields on one page. 
Rendering some forms can take up a couple of seconds (or at least it's very noticeable). 

Unfortunetly it's not possible to split up these huge forms into smaller ones. (It's an internal app and the client needs the forms to be this big, so not much chance there). 

With an event / callback that get's triggered as soon as all form fields are rendered and filled it would allow for some better user experience. For example, it would allow us to show a loading indicator while formly renders the form. 

Another thing where this would be useful would be for custom validations. We have to validate fields based on other fields as soon as the user enters something. So we subscribe to `form.statusChanges` and then check each field. 
This also get's triggered a couple of times during rendering, so it would allow us to pause this custom validation and only start it once everything is finished setting up. 

I also saw that hooks exist for individual fields but as far as I saw this does not support the whole form itself? So [something like this](https://github.com/ngx-formly/ngx-formly/issues/1109#issuecomment-412295111), just for the form?

**Comment by aitboudad** [comment]
> Rendering some forms can take up a couple of seconds (or at least it's very noticeable).

You may consider using virtual scroll https://stackblitz.com/edit/angular-rfwvp6

**Comment by aitboudad** [comment]
Duplicate of #1912
